admin/controllers/check-form-v2.js

```diff
@@ -0,0 +1,64 @@
+const checkFormV2Service = require('../services/check-form-v2.service')
+const ValidationError = require('../lib/validation-error')
+
+const controller = {}
+
+/**
+ * Display initial 'upload and view' check forms page.
+ * @param req
+ * @param res
+ * @param next
+ * @returns {Promise.<*>}
+ */
+controller.uploadAndViewFormsPage = async (req, res, next) => {
+  res.locals.pageTitle = 'Upload and view forms'
+  req.breadcrumbs(res.locals.pageTitle)
+  return res.render('check-form-v2/upload-and-view-forms-v2', {
+    checkForms: [],
+    breadcrumbs: req.breadcrumbs()
+  })
+}
+
+/**
+ * Upload check form view.
+ * @param req
+ * @param res
+ * @param next
+ * @returns {Promise.<void>}
+ */
+controller.uploadCheckFormPage = async (req, res, next) => {
+  req.breadcrumbs('Upload and view forms', '/check-form-v2/upload-and-view-forms-v2')
```

Comment:
I'm not sure about exposing 'v2' in the url.

Reply:
@GeorgeChatzigiannis we don't need both old and new URL's available at runtime, so just add the toggle at the router level

Reply:
@GuyHarwood the toggle is already added at route level similar to other implementations.